InnerEye/ML/reports/classification_report.py

```diff
@@ -176,92 +314,138 @@ def get_metric(val_labels_and_predictions: LabelsAndPredictions,
+    metrics["Sensitivity at optimal threshold"] = 1 - fnr
+
+    return metrics
+
+
+def print_metrics(val_labels_and_predictions: LabelsAndPredictions,
+                  test_labels_and_predictions: LabelsAndPredictions,
+                  is_thresholded: bool = False) -> None:
+    """
+    Given LabelsAndPredictions objects for the validation and test sets, print out some metrics.
+    :param val_labels_and_predictions: LabelsAndPredictions object for the val set. This is used to determine the
+    optimal threshold for classification.
+    :param test_labels_and_predictions: LabelsAndPredictions object for the test set. Metrics are calculated for this
+    set.
+    :param is_thresholded: Whether the model outputs are binary (they have been thresholded at some point)
+                           or are floating point numbers.
+    """
+    metrics = get_all_metrics(val_labels_and_predictions, test_labels_and_predictions, is_thresholded)
+    rows = [[description, f"{value:.4f}"] for description, value in metrics.items()]
+    print_table(rows)
 
 
 def print_metrics_for_all_prediction_targets(val_metrics_csv: Path,
                                              test_metrics_csv: Path,
                                              config: ScalarModelBase,
-                                             is_thresholded: bool = False) -> None:
+                                             val_data_split: Optional[ModelExecutionMode] = None,
+                                             test_data_split: Optional[ModelExecutionMode] = None,
+                                             is_thresholded: bool = False,
+                                             is_crossval_report: bool = False) -> None:
     """
     Given csvs written during inference for the validation and test sets, print out metrics for every prediction target
     in the config.
 
     :param val_metrics_csv: Csv written during inference time for the val set. This is used to determine the
-    optimal threshold for classification. 
+    optimal threshold for classification.
     :param test_metrics_csv: Csv written during inference time for the test set. Metrics are calculated for this csv.
     :param config: Model config
+    :param val_data_split: Whether to filter the validation CSV file for Train, Val, or Test results (default: no
+    filtering).
+    :param test_data_split: Whether to filter the test CSV file for Train, Val, or Test results (default: no filtering).
     :param is_thresholded: Whether the model outputs are binary (they have been thresholded at some point)
                            or are floating point numbers.
-    """
+    :param is_crossval_report: If True, assumes CSVs contain results for multiple cross-validation runs and prints the
+    metrics along with means and standard deviations. Otherwise, prints metrics for a single run.
+    """
+    def get_metrics_for_fold(prediction_target: str, crossval_split: Optional[int] = None) -> Dict[str, float]:
```

Comment:
based on Kristina's confusion about what a "fold" is, maybe we should start calling that a "split" or "cv_split".

Reply:
good catch, missed this one